mypy/test/data.py

```diff
@@ -556,6 +556,9 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    def __init__(self, *, update_data: bool = False) -> None:
```

Comment:
Currently the only test that uses this option is `testcheck`, which uses `update_testcase_output` if it's set. The others simply ignore it.